DSharpPlus/Net/Abstractions/Transport/TransportActivity.cs

```diff
@@ -113,6 +120,38 @@ internal sealed class TransportActivity
         [JsonProperty("assets", NullValueHandling = NullValueHandling.Ignore)]
         public PresenceAssets Assets { get; internal set; }
 
+        /// <summary>
+        /// Gets or sets information about buttons in this rich presence.
+        /// 
+        /// This is a component of the rich presence, and, as such, can only be used by regular users.
+        /// </summary>
+        [JsonProperty("buttons", NullValueHandling = NullValueHandling.Ignore)]
+        public IReadOnlyList<string> Buttons { get; internal set; }
```

Comment:
Same here

Reply:
idk

Reply:
We're never sending this type right? So we don't need to send a Button array. or so we?